lib/miq_automation_engine/service_models/miq_ae_service_user.rb

```diff
@@ -3,6 +3,7 @@ class MiqAeServiceUser < MiqAeServiceModelBase
     expose :current_group,  :association => true
     expose :current_tenant, :association => true
     expose :vms,            :association => true
+    expose :miq_groups,     :association => true
```

Comment:
@jrafanie 
Would this conflict with the new entitlement changes that you are working on?